src/indexer_worker.ts

```diff
@@ -241,11 +261,15 @@ async function main(parentPort: MessagePort, workerData: WorkerData) {
   );
   const indexer = new Indexer(config, network, engine);
   await indexer.start();
+  const blockHeight = (await engine.getBlockHeight()).unwrap() as number;
 
   parentPort
     .on('message', async (blockID: number) => {
       parentPort.postMessage(true); // ack the request
       await indexer.indexBlock(blockID);
+      if (blockID > blockHeight) {
```

Comment:
`blockHeight` is never updated after its initial value is set. Is this intentional?

Reply:
Yes. The reason is that we want to send notifications only for blocks that are coming from [followChainHead](https://github.com/aurora-is-near/aurora-relayer/blob/master/util/indexer/main.go#L85) and having this value initialised when indexer starts seems to be enough.